codegen/src/main/java/software/amazon/awssdk/codegen/poet/common/UserAgentUtilsSpec.java

```diff
@@ -30,12 +32,10 @@
 import software.amazon.awssdk.codegen.poet.PoetExtension;
 import software.amazon.awssdk.codegen.poet.PoetUtils;
 import software.amazon.awssdk.core.ApiName;
-import software.amazon.awssdk.core.util.VersionInfo;
+import software.amazon.awssdk.core.internal.useragent.businessmetrics.BusinessMetricFeatureId;
```

Comment:
This needs to be protected API and moved out of internal package since it's used in generated sdk clients

Reply:
fixed